README.md

```diff
@@ -5,3 +5,81 @@
 Jepsen is a correctness verifcation tools for distributed systems for their
 consistency under special scenarios. This project intended to apply Jepsen on
 Cloud Spanner databases.
+
```

Comment:
Please add a section here on what we achieved in this project. Basically, we should mention how we adopt Jepsen on Spanner and which benchmarks we are supporting.

It should be something similar to https://github.com/jepsen-io/jepsen#design-overview